lib/go-atscfg/facts.go

```diff
@@ -39,13 +41,13 @@ func Make12MFacts(
 	}
 	warnings := []string{}
 
-	if server.Profile == nil {
+	if server.ProfileNames == nil && len(server.ProfileNames) == 0 {
 		return Cfg{}, makeErr(warnings, "this server missing Profile")
 	}
 
 	hdr := makeHdrComment(opt.HdrComment)
 	txt := hdr
-	txt += "profile:" + *server.Profile + "\n"
+	txt += "profile:" + strings.Join(server.ProfileNames, ",") + "\n"
```

Comment:
The header comment is meant to be human readable, so grammar and spacing count. `profile:` should probably be `profiles:` now that there are multiple, and the Profile names should be joined with `", "` to make reading the list easier.